### ISSUE TITLE
Descriptions of columns are not exported  MS SQL Server

### ISSUE THREAD

**Issue by PhillipTommerholt** [keeps/db-preservation-toolkit#314]
Beta 7.2
MS SQL Server
I have a Northwind database where I have added a  danish description to a column by executing this command:

```
EXEC sp_addextendedproperty 
    @name = N'MS_Description', @value = 'Kategoriens navn',
    @level0type = N'Schema',   @level0name = 'dbo',
    @level1type = N'Table',    @level1name = 'Categories',
    @level2type = N'Column',   @level2name = 'CategoryName';
GO 
```

I can see the extended property in SQL Management Studio:
![image](https://cloud.githubusercontent.com/assets/15984159/26394276/aed91068-406c-11e7-86df-601f38bf1ca5.png)

When I import the database and export to SIARDDK and SIARD2 this description is not exported. 
I have tried it the other way around to import a SIARDDK and a SIARD2 package and export it to MS SQL Server - again descriptions are not exported.

**Comment by chalkos** [comment]
Related to #174

**Comment by luis100** [comment]
@chalkos is this solved?

**Comment by chalkos** [comment]
Yes. Descriptions on SQL Server are supported via an `extendedproperty` named `MS_Description`

**Comment by PhillipTommerholt** [comment]
@chalkos @luis100 
I just ran  a sql server siard-2 export where descriptions are exported with 2.3.0 RC.
I thereafter imported the siard-file and exported it to another database on SQL Server, and here the descriptions were not exported.

**Comment by chalkos** [comment]
What I gathered was that the descriptions were not being migrated from SQL Server to SIARD. This is now fixed, hence closing the issue.

Migrating the descriptions from SIARD to a DBMS is far from our focus (it has similarities with https://github.com/keeps/db-preservation-toolkit/issues/177#issuecomment-486157045 and https://github.com/keeps/db-preservation-toolkit/issues/11#issuecomment-486164332).
Our focus being that the database is preserved and the information can be accessed (eg: via the database visualization toolkit), which is the case, so I considered this issue to be resolved.
